### ISSUE TITLE
Possibility to use as chunk ?

### ISSUE THREAD

**Issue by Ziker22** [webpack-contrib/i18n-webpack-plugin#67]
Hi guys,
First of all thanks for great work.
I have a question though. 
In my app the only way to determine which language is being used is to visit login screen then let user select lang value in combobox then hit login and after that im 100% sure about locale (it can be used in app you have to logout/login again)
So my question is . Is there some way to code split my app to login and everything else and then load lets say bundle_de_DE.js after button is clicked ?

**Comment by davidfurlong** [comment]
Yes. But you'll probably want to check out the code splitting docs, this isn't relevant to this package.
